### PR TITLE
Implemented System Parameters Page

### PR DESCRIPTION
**Pull request by vedangimittal**
- Implemented System Parameters Page - 1060
- Jira Story: https://jsw.ibm.com/browse/PFEBMC-2390

### CODE REVIEW

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]
Getting success when we have empty values in "RPD scheduled run time of day". It should throw error

File: src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue [hunk 1]
```diff
@@ -172,237 +172,229 @@
...
+const options = computed(() => {
+  if (systemParametersStore.rpdPolicyOptionsGetter){
+  return systemParametersStore.rpdPolicyOptions.map((option) => ({
+    value: option,
+    text: option,
+  }));
+ } else return;
+});
+const rpdFeatOptions = computed(() => {
+  console.log('systemParametersStore.rpdFeatureOptionsGetter', systemParametersStore.rpdFeatureOptionsGetter);
```
Comment (nabilpandiyalayil) [inline src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue]: Remove console log

File: src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue [hunk 2]
```diff
@@ -172,237 +172,229 @@
...
+}));
+
+const v$ = useVuelidate(rules, { rpdScheduledRun, rpdScheduledRunDuration });
+
+// watch(selectedItem, (newValue) => {
+//   systemParametersStore.rpdPolicy = newValue;
+// });
+// watch(selectedFeatureItem, (newValue) => {
+//   systemParametersStore.rpdFeature = newValue;
+// });
```
Comment (nabilpandiyalayil) [inline src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue]: I think these watchers should be enabled.

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

File: src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue (hunk 2 above)
Comment (vedangimittal) [reply src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue]: selectedItem and selectedFeatureItem do not exist in any file

**Review by Nikhil-Ashoka — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Pull request merged**